packages/providers/package.json

```diff
@@ -13,23 +13,23 @@
     "net": "./lib/browser-net.js"
   },
   "dependencies": {
-    "@ethersproject/abstract-provider": "^5.5.0",
-    "@ethersproject/abstract-signer": "^5.5.0",
-    "@ethersproject/address": "^5.5.0",
-    "@ethersproject/basex": "^5.5.0",
-    "@ethersproject/bignumber": "^5.5.0",
-    "@ethersproject/bytes": "^5.5.0",
-    "@ethersproject/constants": "^5.5.0",
-    "@ethersproject/hash": "^5.5.0",
-    "@ethersproject/logger": "^5.5.0",
-    "@ethersproject/networks": "^5.5.0",
-    "@ethersproject/properties": "^5.5.0",
-    "@ethersproject/random": "^5.5.0",
-    "@ethersproject/rlp": "^5.5.0",
-    "@ethersproject/sha2": "^5.5.0",
-    "@ethersproject/strings": "^5.5.0",
-    "@ethersproject/transactions": "^5.5.0",
-    "@ethersproject/web": "^5.5.0",
+    "@hethers/abstract-provider": "1.0.0",
+    "@hethers/abstract-signer": "1.0.0",
+    "@hethers/address": "1.0.0",
+    "@ethersproject/basex": "5.5.0",
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@hethers/constants": "1.0.0",
+    "@ethersproject/hash": "5.5.0",
+    "@hethers/logger": "1.0.0",
+    "@hethers/networks": "1.0.0",
+    "@ethersproject/properties": "5.5.0",
+    "@ethersproject/random": "5.5.0",
+    "@ethersproject/rlp": "5.5.0",
```

Comment:
We do not need the rlp package. Maybe we should remove it

Reply:
`rlp` package is removed